Project.toml

```diff
@@ -6,6 +6,10 @@ version = "0.1.0"
 
 [deps]
 JSON = "682c06a0-de6a-54ab-a142-c8b1cf79cde6"
+LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 Plots = "91a5bcdd-55d7-5caf-9e0b-520d859cae80"
 QuantumAnnealing = "4832667a-bab9-40a8-88f6-be9efce3ea89"
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
+
+[compat]
+QuantumAnnealing = "0.1"
```

Comment:
I think this will work but in an abundance of caution you might choose to use `~0.1` instead.  Review these docs to know for sure, https://pkgdocs.julialang.org/v1/compatibility/